build.bat

```diff
@@ -732,73 +854,75 @@ GOTO:EOF
+	echo.                           	  (No effect when building for Micro)
+	echo.    --cpp-build            	  Only C++ code generation and compilation.
+	echo.    --cpp03-build          	  Only C++ New PSM code generation and
+	echo.                           	  compilation.
+	echo.                           	  (No effect when building for Micro)
+	echo.    --cs-build             	  Only C Sharp code generation and compilation.
+	echo.                           	  (No effect when building for Micro)
+	echo.    --cmake  path          	  Path to the CMAKE executable. If this
+	echo.                           	  parameter is not present, Cmake variable
+	echo.                           	  should be available from your $PATH variable.
+	echo.    --cmake-generator g    	  CMake generator to use. By default, NMake
+	echo.                           	  makefiles will be generated.
+	echo.    --perl path            	  Path to PERL executable. If this parameter is
+	echo.                           	  not present, the path to PERL should be
+	echo.                           	  available from your \$PATH variable.
+	echo.    --java-home path       	  Path to the Java JDK home folder. If this
+	echo.                           	  parameter is not present, javac, jar and java
+	echo.                           	  executables should be available from your
+	echo.                           	  $PATH variable.
+	echo.                           	  (No effect when building for Micro)
+	echo.    --debug                	  Compile against the RTI Connext Debug
+	echo.                           	  libraries. Default is against release ones.
+	echo.    --dynamic                    Compile against the RTI Connext Dynamic
+	echo.                           	  libraries. Default is against static ones.
+	echo.                           	  (No effect when building for Micro)
+	echo.    --secure               	  Enable the security options for compilation.
+	echo.                           	  Default is not enabled.
+	echo.    --openssl-home path    	  Path to the openssl home. This will be used
+	echo.                           	  when compiling statically and using security
+	echo.                           	  Note: For Micro provide this path with /
+	echo.                           	  instead of \, required by cmake.
+	echo.    --clean                	  If this option is present, the build.sh script
+	echo.                           	  will clean all the generated code and binaries
+	echo.                           	  from previous executions.
+	echo.    --customType type      	  Use the Custom type feature with your type.
+	echo.                           	  See details and examples of use in the
+	echo.                           	  documentation.
+	echo.    --flatdata-max-size size     Specify the maximum bounded size in bytes      
+    echo.                                 for sequences when using FlatData language     
```

Comment:
Indentation is done with tabs, but separation for the echos with spaces.

Reply:
Done